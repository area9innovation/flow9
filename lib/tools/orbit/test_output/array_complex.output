Flow compiler (3rd generation)

Processing 'orbit' on http server

Variable not found: b
Variable not found: b
Result: [6]
