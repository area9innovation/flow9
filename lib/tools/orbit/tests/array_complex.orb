// More complex array tests

// Arrays with expressions
[1 + 2, 3 * 4, 5 / 2];

// Mixed type arrays (if supported)
[1, "hello", true];

// Array with function call result
fn add(a, b) = a + b;
[add(1, 2), add(3, 4)];

// Using arrays in expressions
let arr1 = [1, 2, 3];
let arr2 = [4, 5, 6];
arr1;
arr2;