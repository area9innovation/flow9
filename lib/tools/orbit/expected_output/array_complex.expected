Flow compiler (3rd generation)

Processing 'orbit' on http server

Result: [4, 5, 6]
VERIFY - THIS OUTPUT NEEDS HUMAN VERIFICATION
